config/Migrations/20200828235617_CreateTagsLinks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+use Migrations\AbstractMigration;
+
+class CreateExports extends AbstractMigration
```

Comment:
You need to rename the class to `CreateTagsLinks` because `CreateExports` already exists.